### PR TITLE
Player movement

### PR DESCRIPTION
**Pull request by cam-shotter**
Hudson won the world

### CODE REVIEW

File: client/actions/playerMovement.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    type: "PLAYER_MOVE_UP"
+  }
+}
+const playerMoveDown = () => {
+  return {
+    type: "PLAYER_MOVE_DOWN"
+  }
+}
+
+// bindActionCreators({
```
Comment (DylanHul) [inline client/actions/playerMovement.js]: remove please 📦  🎊

File: client/components/game.js [hunk 2]
```diff
@@ -5,6 +5,26 @@ class Game extends React.Component {
 
   constructor(props){
     super(props)
+
+  }
+
+  componentDidMount() {
+    document.addEventListener("keydown", (key) => {
```
Comment (DylanHul) [inline client/components/game.js]: change (key) to (e) 🍍  🍸

File: client/components/player.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ class Player extends React.Component {
   render(){
     return (
       <div className='player'>
-        <span><p>{this.props.player.health}</p></span>
+        <span><p>{this.props.x}</p></span>
```
Comment (DylanHul) [inline client/components/player.js]: change to something with font awesome 👔

File: client/actions/playerMovement.js (hunk 1 above)
Comment (cam-shotter) [reply client/actions/playerMovement.js]: Done 🌜

**Comment by cam-shotter** [comment]
Fixed merge comments :neckbeard:

**Pull request merged**